docs/sources/alerting/fundamentals/notifications/_index.md

```diff
@@ -95,21 +100,21 @@ The notification policy tree is responsible for:
 Each notification policy handles specific tasks:
 
 - Deciding which contact point receives the alert notification.
-- Controlling when to send notifications based on its [notification timings](ref:notification-timings).
-- [Grouping multiple alerts](#group-alert-notifications) into a single notification to reduce alert noise.
+- Controlling when to send notifications based on its notification timing options.
+- Grouping multiple alerts into a single notification to reduce alert noise.
 
-{{< figure src="/media/docs/alerting/alerting-notification-policy-diagram-v2.png" max-width="750px" alt="A diagram of the notification policy component" >}}
+{{< figure src="/media/docs/alerting/alerting-notification-policy-diagram-v5.png" max-width="750px" alt="A diagram of the notification policy component" >}}
```

Comment:
havent checked-but is this just a screenshot of the ui? :) if so, take it out:)

Reply:
No, the image is not a screenshot of the UI.

<img width="862" alt="alerting-notification-policy-diagram-v5" src="https://github.com/grafana/grafana/assets/825430/79b53b6a-783c-4c10-b20d-049ac953a4cf">